bundles/org.smarthomej.binding.tuya/README.md

```diff
@@ -117,6 +117,63 @@ The `min` and `max` parameters define the range allowed (e.g. 0-86400 for turn-o
 The `string` channel has one additional (optional) parameter `range`.
 It contains a comma-separated list of command options for this channel (e.g. `white,colour,scene,music` for the "workMode" channel).
 
+### Type `ir-code`
+
+IR code types:
++ `Tuya DIY-mode` - use study codes from real remotes. 
+   
+   Make a virtual remote control in DIY, learn virtual buttons.
+
++ `Tuya Codes Library (check Advanced options)` - use codes from templates library. 
+   
+   Make a virtual remote control from pre-defined type of devices.
+
+   Select Advanced checkbox to configure other parameters: 
+   + `irCode` - Decoding parameter
+   + `irSendDelay` - used as `Send delay` parameter
+   + `irCodeType` - used as `type library` parameter
+
++ `NEC` - IR Code in NEC format
++ `Samsung` - IR Code in Samsung format.
+
+**Additional options:**
+* `Active Listening` - Device will be always in learning mode. After send command with key code device stays in the learning mode
```

Comment:
```suggestion
* `Active Listening` - Device will be always in learning mode.
After send command with key code device stays in the learning mode
```

Always start a new sentence on a new line. This will greatly improve reviewing future changes. (also in other places)